Marlin/src/module/probe.cpp

```diff
@@ -684,6 +695,15 @@ float Probe::probe_at_point(const float &rx, const float &ry, const ProbePtRaise
   // Move the probe to the starting XYZ
   do_blocking_move_to(npos);
```

Comment:
I've found that disabling the probe needs to happen before moving in the XY direction:

https://github.com/CR6Community/Marlin/commit/500cbe4cba46371bc1a9e263953f6d1e5185da21

Reply:
Brought over more of your suggested changes.